### PR TITLE
Add epidemic risk vignette and `probability_contain()`

### PR DESCRIPTION
**Pull request by joshwlambert**
This PR addresses #12 (and makes a start on #13). 

The primary focus of this PR is to add a vignette that discusses the application of {superspreading} functions to decision making and policy. This takes inspiration from the document initially shared in #5 and mentioned in #12 which is a preprint used by SAGE during the COVID-19 pandemic.

As part of this vignette on control measures, a new function was added: `probability_contain()`. This implements population-level control measures. It does not complete the requirements of #13 which also requires individual level control and may relate to development from epiverse-trace/epichains#3.

Tests and documentation are added for `probability_contain()`. {bpmodels} is added as a remote dependency, and {ggtext} as a suggested dependency.

### CODE REVIEW

File: R/probability_contain.R [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#'
+#' @references
+#'
+#' Lloyd-Smith, J. O., Schreiber, S. J., Kopp, P. E., & Getz, W. M. (2005)
+#' Superspreading and the effect of individual variation on disease emergence.
+#' Nature, 438(7066), 355-359. <https://doi.org/10.1038/nature04153>
+#'
+#' @examples
+#' probability_contain(R = 1.5, k = 0.5, c = 1)
+probability_contain <- function(R, k, a = 1, c, # nolint
```
Comment (jamesmbaazam) [inline R/probability_contain.R:27]: Please document `a`.

Also include checks for `0 =< c <= 1`.

File: vignettes/epidemic_risk.Rmd [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+During the early stages of an infectious disease outbreak information on transmission
+clusters can provide insight into the occurrence of superspreading events and from this
```
Comment (jamesmbaazam) [inline vignettes/epidemic_risk.Rmd:25]: I suggest to define "transmission clusters" and "superspreading events". This is a good place for beginners to learn about the subject.

Comment (jamesmbaazam) [inline vignettes/epidemic_risk.Rmd:25]:
```suggestion
clusters and heterogeneity in individual-level transmission can provide insight into the occurrence of superspreading events and from this
```

**Review by jamesmbaazam — CHANGES_REQUESTED** [review]
This is a nice vignette to illustrate a use case for the package. I've suggested a few changes that should be straightforward to fix.

File: R/probability_contain.R (hunk 1 above)
Comment (joshwlambert) [reply R/probability_contain.R:27]: `a` is documented by inheriting from `probability_epidemic()`

Comment (joshwlambert) [reply R/probability_contain.R:27]: Resolved in b971d7b237888ea6792368a1b31e26ff4a39c926

File: vignettes/epidemic_risk.Rmd (hunk 2 above)
Comment (joshwlambert) [reply vignettes/epidemic_risk.Rmd:31]: Added link to getting started vignette which includes more introductory content and definitions in ff0b6f2753a6ed75107f41a9610816bd0c1caf9d

File: vignettes/epidemic_risk.Rmd [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+  theme(
+    axis.title.x = element_markdown(),
+    legend.title = element_markdown()
+  )
+```
+
+## Superspreading in decision making
+
+One of the first tasks in an outbreak is to establish whether estimates of individual-level
+transmission variability have been reported, either for this outbreak or a previous outbreak
```
[the remark this answers is not in the document]
Comment (joshwlambert) [reply vignettes/epidemic_risk.Rmd:215]: At least from what I've seen from COVID, MERS, Ebola and other recent outbreaks, this sort of analysis is routine.

File: vignettes/epidemic_risk.Rmd [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+  annotate(geom = "text", x = 0.45, y = 0.75, label = "MERS") +
+  scale_y_continuous(
+    name = "Probability of large outbreak",
+    limits = c(0, 1)
+  ) +
+  scale_x_continuous(name = "Extent of homogeneity in transmission") +
+  theme_bw()
+```
+
+The degree of variability in transmission increases as $k$ decreases. So the probability
```
[the remark this answers is not in the document]
Comment (joshwlambert) [reply vignettes/epidemic_risk.Rmd]: Add citation in 32cf43213e2a172584039b26bd27b08762b64428

**Comment by joshwlambert** [comment]
Thanks for the comments @jamesmbaazam!

**Review by jamesmbaazam — APPROVED** [review]

**Pull request merged**